src/TgBotLib.Core/Interfaces/IUsersActionsService.cs

```diff
@@ -4,7 +4,7 @@ namespace TgBotLib.Core;
 
 public interface IUsersActionsService
 {
-    void HandleUser(long chatId, string actionName);
+    void HandleUser(long chatId, string actionName, object model);
```

Comment:
Давай переименуем тут и ниже:
```suggestion
    void HandleUser(long chatId, string actionName, object payload);
```